server/book/book.go

```diff
@@ -50,8 +50,9 @@ func New(lotSize uint64, halfCapacity ...uint32) *Book {
 // Clear reset the order book with configured capacity.
 func (b *Book) Clear() {
 	b.mtx.Lock()
+	b.buys, b.sells = nil, nil
 	b.buys = NewMaxOrderPQ(b.halfCap)
-	b.sells = NewMaxOrderPQ(b.halfCap)
+	b.sells = NewMinOrderPQ(b.halfCap)
```

Comment:
This function only happens on book purge with market suspend, so if the op resumed without restarting the whole thing, it would be backwards on the sell side.

The reallocation was seen to press system memory since the PQ is the bulk of the allocated memory for dcrdex, so I cut the default book capacity in half and nil'd both queue variables before allocating new ones in case that helps Go GC through it.